### PR TITLE
Fix typo in pi_vae.py

### PR DESCRIPTION
**Pull request by eltociear**
implmentation -> implementation

### CODE REVIEW

**Comment by eltociear** [comment]
@cla-bot check

**Comment by stes** [comment]
Thanks for the contribution, @eltociear --- just fyi, the PR looks fine, the build errors are unrelated to the PR and related to the workflow config. I will look into it and merge this, no further action required on your end.

**Comment by stes** [comment]
Looking into this. Two issues:

- For the documentation status, the PR [does not have permission to comment on the thread](https://github.com/AdaptiveMotorControlLab/CEBRA/actions/runs/4967556431/jobs/8898075689?pr=20)
- For the build docs action, the GITHUB_TOKEN to checkout the cebra-figures is [not valid/cannot be found](https://github.com/AdaptiveMotorControlLab/CEBRA/actions/runs/4967556430/jobs/8898083794?pr=20).

In both cases it seems like adding PATs will fix the issue, I will go ahead and try this now.

File: .github/workflows/doc-coverage.yml [hunk 1]
```diff
@@ -8,7 +8,11 @@ name: PR Status
 # opens a PR.
 # IMPORTANT: DO NOT EXPOSE REPOSITORY SECRETS WITHIN THIS PR!
 on:
-  pull_request:
+  # NOTE(stes): Use pull_request_target instead of pull_request to allow
+  # to post comments on the current PR, even when an external contributor
+  # opens a PR.
+  # IMPORTANT: DO NOT EXPOSE REPOSITORY SECRETS WITHIN THIS PR!
+  pull_request_target:
```
Comment (stes) [inline .github/workflows/doc-coverage.yml:15]:
```suggestion
  pull_request:
```

**Comment by stes** [comment]
@MMathisLab should pass now, lets see.

**Review by stes — APPROVED** [review]
When merging, let's remove the additional description text from the intermediate commits we did here.

**Pull request merged**